dask_cuda/tests/test_explicit_comms.py

```diff
@@ -176,6 +186,11 @@ def test_dataframe_shuffle(backend, protocol, nworkers, _partitions):
     if backend == "cudf":
         pytest.importorskip("cudf")
 
+    if QUERY_PLANNING_ON:
+        # There seem to be problems with dask_expr<1.0 and dask<2024.2.1
+        pytest.importorskip("dask", minversion="2024.2.1")
+        pytest.importorskip("dask_expr", minversion="1.0")
```

Comment:
**We should not merge this.** This is mostly a note/demonstration that the problem has to do with a bad/old dask + dask-expr combination in CI. Since dask is "unpinned", we should definitely be getting dask_expr>1.0, but the lack of nightly packages for dask_expr can produce a bad combination for us.